File: Python3/프로그래머스/0/181930. 주사위 게임 2/주사위 게임 2.py
def solution(a, b, c):
    answer = a + b + c
    
    if a == b or b == c or a == c: 
        answer *= pow(a, 2) + pow(b, 2) + pow(c, 2)
        
    if a == b and b == c: 
        answer *= pow(a, 3) + pow(b, 3) + pow(c, 3)
        
    return answer

File: Swift/프로그래머스/0/181930. 주사위 게임 2/주사위 게임 2.swift
import Foundation

func solution(_ a:Int, _ b:Int, _ c:Int) -> Int {
    var result = a + b + c
    
    if a == b || b == c || a == c {
        result *= Int(pow(Double(a), 2.0) + pow(Double(b), 2.0) + pow(Double(c), 2.0))
    }
    
    if a == b && b == c {
        result *= Int(pow(Double(a), 3.0) + pow(Double(b), 3.0) + pow(Double(c), 3.0))
    }
    
    return result
   
}